File: AppPedido/src/app/Lista.ts
import { Mesa } from '../app/Mesa';
import { Desconto } from '../app/Desconto';
import { Taxa } from '../app/Taxa';
import { Pagamento } from '../app/Pagamento';
import { Garcom } from '../app/Garcom';
import { Produto } from '../app/Produto';
import { Cliente } from '../app/Cliente';
export class Lista {
  id: number = 0;
  status: string = '';
  somaTotal: number = 0;
  mesa: Mesa | undefined;
  desconto: Desconto | undefined;
  taxa: Taxa | undefined;
  pagamento: Pagamento | undefined;
  garcom: Garcom | undefined;
  produtos: Produto | undefined;
  cliente: Cliente | undefined;
}

File: WebApplicationPedido/Models/Lista.cs
using System.ComponentModel.DataAnnotations;

namespace Pedido.Models
{
    public class Lista
    {
        [Key]
        public int Id { get; set; }
        public string? Status { get; set; }
        public float? SomaTotal { get; set; }
        public Mesa? Mesa { get; set; }
        public Desconto? Desconto { get; set; }
        public Taxa? Taxa { get; set; }
        public Pagamento? Pagamento { get; set; }
        public Garcom? Garcom { get; set; }
        public List<Produto>? Produtos { get; set; }
        public Cliente? Cliente { get; set; }
    }
}